src/main/java/org/sagebionetworks/template/etl/EtlConfigValidator.java

```diff
@@ -31,8 +31,8 @@ private void validateStream(EtlDescriptor etlDescriptor) {
         if (StringUtils.isBlank(etlDescriptor.getSourcePath())) {
             throw new IllegalStateException("The etl s3 source path cannot be empty");
         }
-        if (StringUtils.isBlank(etlDescriptor.getDestinationPath())) {
-            throw new IllegalStateException("The etl s3 destination path cannot be empty");
+        if (StringUtils.isBlank(etlDescriptor.getDestinationBucket())) {
```

Comment:
The destination buckt should be at the higher level (e.g. not for each job) or hardcoded in the template